### PR TITLE
My Submissions view #22

### PR DESCRIPTION
**Pull request by Anto8819**
Screenshot 2022-09-28 at 13 38 39
Screenshot 2022-09-28 at 13 45 57
Screenshot 2022-09-28 at 13 45 41

### CODE REVIEW

File: applications/portal/frontend/src/theme/Theme.tsx [hunk 1]
```diff
@@ -29,10 +29,20 @@ const theme = createTheme({
       main: "#2173F2",
       light: "#DFEBF8",
       dark: "#0052CC",
-      200: "#A5C8F0"
+      200: "#A5C8F0",
```
Comment (filippomc) [inline applications/portal/frontend/src/theme/Theme.tsx:32]: 200 is a strange name for a color

File: applications/portal/frontend/src/App.tsx [hunk 2]
```diff
@@ -21,7 +21,9 @@ const App = () => {
         <CssBaseline />
         <Navbar />
         <Switch>
-          <Route exact path="/" component={Home} />
+          <Route exact path="/">
+            <Home activeTab="all results" />
```
Comment (filippomc) [inline applications/portal/frontend/src/App.tsx:25]: It would be better to use constants for tab names

**Review by filippomc — COMMENTED** [review]
Looks good, left a few minor refactoring comments

File: applications/portal/frontend/src/theme/Theme.tsx (hunk 1 above)
Comment (Anto8819) [reply applications/portal/frontend/src/theme/Theme.tsx]: done, changed to contrastText

File: applications/portal/frontend/src/App.tsx (hunk 2 above)
Comment (Anto8819) [reply applications/portal/frontend/src/App.tsx]: done

**Review by filippomc — APPROVED** [review]

**Pull request merged**